main.tf

```
@@ -68,7 +107,8 @@ resource "aws_iam_role" "bkup_role" {
           }
         }
       ]
-  })
+    }
+  )
```

```suggestion
    }
  })
```
looks like an accidental change